### DISCUSSION TITLE
Calendar Support

### DISCUSSION THREAD

**Discussion by avow1337** [sct/overseerr#473] [Feature Requests]
Would it ever be possible to bring the calendars from Sonarr/Radarr into Overseerr to expose them externally?

**Comment by Weavus** [comment]
Yeah, thats a really nice idea...

**Comment by marksie1988** [comment]
im also looking for this feature, I'm trying to build something with the API and need a calendar endpoint that allows me to get upcoming TV  or movies. 

something like this:

`calendar?start='date'&end='date'&mediaType='all|tv|movies'`

should return the `MediaInfo` schema.
